.rubocop.yml

```diff
@@ -1,29 +1,169 @@
+inherit_from:
+  - .rubocop_todo.yml
+
 AllCops:
+  TargetRubyVersion: 2.1
+  TargetRailsVersion: 3.2
   Include:
     - '**/Rakefile'
     - '**/config.ru'
   Exclude:
     - 'db/**/*'
     - 'config/**/*'
     - 'script/**/*'
-    - 'spec/**/*'
     - !ruby/regexp /old_and_unused\.rb$/
 
-Documentation:
+# OFN SETTINGS
+# Cop settings that have been agreed upon by the OFN community
+
+Rails:
+  Enabled: true
+
+Style/Documentation:
+  Enabled: false
+
+Style/StringLiterals:
+  Enabled: false
+
+# TEMPORARY/CONTESTED SETTINGS
+# These are still to be decided upon, but recommended for inclusion by
+# oeoeaio after scrutinising offenses the codebase
+
+# Don't think this is a big issue, mostly picking up RPSEC scope definitions
+# with lamdas and RSpec '.to change{}' blocks
+Lint/AmbiguousBlockAssociation:
+  Enabled: false
+
+# No alternative to this one until we upgrade to Rails 4 and can use #find_by
+Rails/DynamicFindBy:
+  Enabled: false
+
+# This should be the programmer's discretion, perhaps we should review all of
+# the uses of it an make specific exceptions though.
+Rails/SkipsModelValidations:
+  Enabled: false
+
+# This should be enfored, but doing so would require changing the entire
+# file, so should probably do when outstanding PRS are at a minimum
+Style/ClassAndModuleChildren:
```

Comment:
I'm not sure that this is actually how Rubocop itself works is it? I think that the exceptions listed in `rubocop_todo.yml` are treated in the same way as those in `rubocop.yml`, having the two files just provides a way of separating out existing errors that have not been looked at yet (`rubocop_todo.yml`) from agreed upon configurations (`rubocop.yml`). So if we are using Rubocop linters, and we remove this line, devs will start getting errors about things that they didn't cause. Not sure if that is a problem or not...

From my understanding, CodeClimate adds an additional layer of magic, in that it understands git and only raises Rubocop errors if they are not present on the base branch (ie. `master`). That is great, but as above, doesn't help us with linters.